src/metrics/build-metric-log.ts

```diff
@@ -0,0 +1,10 @@
+// Builds the string representation of the metric that will be written to logs
+export function buildMetricLog(name: string, value: number, tags: string[]) {
+  return `${JSON.stringify({
+    // Date.now() returns Unix time in milliseconds, we convert to seconds for DD API submission
+    e: Date.now() / 1000,
```

Comment:
Should this be rounded?

Reply:
the decimals don't seem to make a difference, in testing the metrics are ingested ok with the decimal timestamps